frontend/packages/kubevirt-plugin/src/components/vm-disks/disk-summary.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import {
+  getContainerImageByDisk,
+  getURLSourceByDisk,
+  getPVCSourceByDisk,
+  getWindowsToolsURLByDisk,
+} from '../../selectors/vm/selectors';
+
+const getDiskSource = (vm, { name }) => {
+  const container = getContainerImageByDisk(vm, name);
+  const windowsToolsURL = getWindowsToolsURLByDisk(vm, name);
+  const pvc = getPVCSourceByDisk(vm, name);
+  const url = getURLSourceByDisk(vm, name);
+
+  if (windowsToolsURL) return <li key={name}>Windows Tools - {windowsToolsURL}</li>;
```

Comment:
instead of these for returns i'd use switch and one return statement.

Reply:
switch on what? we need to check every variable ( only one of them is not null at a time )

Reply:
when you use https://github.com/kubevirt/web-ui-components/blob/master/src/selectors/dv/selectors.js#L17 as mentioned above you can do

```javascript
const diskSource = getDataVolumeSourceType(vm, name);
let summary: string;
switch(diskSource.type) {
  case DATA_VOLUME_SOURCE_URL:
    summary = `URL - ${disk.url}`;
  case DATA_VOLUME_SOURCE_PVC:
    summary = `PVC - ${diskSource.name}`;
 ...
}
return <li key={name}>{summary}</li>
```